ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/ConversionsSpec.scala

```diff
@@ -105,122 +109,226 @@ class ConversionsSpec extends AnyWordSpec with Matchers with OptionValues {
       "convert rejection to proto models and back to expected grpc code" in {
         forAll(
           Table(
-            ("rejection", "expected code"),
+            ("rejection", "expected code", "expected additional details"),
             (
               Rejection.ValidationFailure(Error.Package(Error.Package.Internal("ERROR", "ERROR"))),
               Code.INVALID_ARGUMENT,
+              Map.empty[String, String],
```

Comment:
You may be able to remove `[String, String]` from this line and every other if you add types to this `Table` too.

Reply:
Yes, you are right, forgot about this.